taui/src/index.js

```diff
@@ -76,6 +76,7 @@ function mapStateToProps (state, ownProps) {
     activeListing: get(state, 'data.activeListing'),
     activeNetworkIndex: select.activeNetworkIndex(state, ownProps),
     activeNeighborhoodBounds: select.activeNeighborhoodBounds(state, ownProps),
+    authToken: select.authToken(state, ownProps),
```

Comment:
These selectors are so simple that I'd suggest just inlining the `get()` call right here, like is done on line 76, above:
```suggestion
    authToken: get(state, 'authToken'),
```

That should then let us get rid of the separate files for these selectors.